packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -230,7 +224,7 @@ export class Relayer {
     }
 
     // Fetch deposit for relay.
-    const deposit = await this.matchRelayWithDeposit(relay);
+    const deposit = await this._matchRelayWithDeposit(relay);
```

Comment:
I did a fair bit of re-naming as well for consistency with the rest of this file.